entrypoint.sh

```diff
@@ -24,22 +24,15 @@
 
 
 if [ "$1" = "test" ]; then
-   pip --quiet install microcosm-metrics
-   # Install standard test dependencies; YMMV
-   pip --quiet install \
-       .[test]
-   pytest
+   pip install --no-cache-dir --upgrade --extra-index-url ${EXTRA_INDEX_URL} .\[test\]
+   shift
+   exec pytest "$@"
```

Comment:
should not we use codeartifact here? and what's the `shift` for?

Reply:
fixed